assets/agenda/utils.ts

```diff
@@ -186,6 +186,14 @@ export function getCoverageDisplayName(coverageType: any) {
         get(coverageTypes, `${coverageType}.name`, coverageType);
 }
 
+export function getCoverageAsigneeName(coverage: any) {
```

Comment:
Please use types, `any` should **not** be used in new code, this includes function arguments and return type.